### PR TITLE
added all homework tasks

### PR DESCRIPTION
**Pull request by rajanibusani**
js1/wk1 hw

### CODE REVIEW

File: javascript/javascript1/week1/index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+console.log(stringText)
+
+//TASK2: A dog age calculator
+
+const dogYearOfBirth = 2015; //fixed. No assignment of new values.
+let dogYearFuture = 2025; //can assign and check with other years.
+let dogYear = dogYearFuture-dogYearOfBirth;
+
+let shouldShowResultInDogYears;
+if(shouldShowResultInDogYears == true){
```
Comment (RaMin0) [inline javascript/javascript1/week1/index.js:21]: `shouldShowResultInDogYears` is expected to have the type `boolean`. Comparing it to `true` or `false` can be replaced with the variable itself as the condition.

File: javascript/javascript1/week1/index.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+//Peters house price
+let volumeInMeters1 = widthOfHouse[0]*heightOfHouse[0]*depthOfHouse[0];
+const amtPayingByPeter = 2500000;
+let priceOfPetersHouse;
+priceOfPetersHouse = volumeInMeters1 * 2.5 * 1000 + gardenSizeInM2[0] * 300;
+
+if (priceOfPetersHouse <= amtPayingByPeter){
+    console.log(`Actual price of the house is ${priceOfPetersHouse}.\nPeter is paying (${amtPayingByPeter}) too high for the house.`)
+    
+}
```
Comment (RaMin0) [inline javascript/javascript1/week1/index.js:48]: If Peter is paying the right amount for the house, will the logged message make sense? What do you think?

File: javascript/javascript1/week1/index.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+} 
+else {
+    console.log(`Julia is paying (${amtPayingByJulia}) too little for the house.`) 
+}
+
+//Task4: Ez Namey (Startup name generator)
+
+let firstWords = ['Easy','Awesome','Corporate','Trade','Creativ','Software','Computer','Digital','Professional','Innovative'];
+let secondWords = ['Developers', 'Tech', 'Services','Solutions','Systems','Development','Tech Tools','Technologies','Group','Technologies'];
+const randomNumber = Math.floor(Math.random() * 10);
```
Comment (RaMin0) [inline javascript/javascript1/week1/index.js:70]: How about using 2 random indices; one for each array? Will this allow for more combinations of startup names to be generated?

**Review by RaMin0 — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week1/index.js (hunk 3 above)
Comment (rajanibusani) [reply javascript/javascript1/week1/index.js]: Thank you so much for your time and suggestions..! i made the changes and can i merge the pull request now..tq

Comment (RaMin0) [reply javascript/javascript1/week1/index.js]: You're welcome :)

**Review by RaMin0 — APPROVED** [review]

**Pull request merged**